### PR TITLE
Refine some configurations in TSN model

### PR DESCRIPTION
**Pull request by LiuChaoXD**
the refined tsn model based on the paddle developed platform

### CODE REVIEW

**Comment by LiuChaoXD** [comment]
Refine the batch size and learning rate settings for TSN model
1. For the single gpu runner, I rewrite the settings about batch size and learning rate.
2. Add a shell command file for downloading the  pretrained weights
3. Rephrase the readme for conveniently reading

**Review by shippingwang — APPROVED** [review]
